pycln/utils/scan.py

```diff
@@ -358,8 +358,16 @@ def handle_c_libs_importables(module_name: str) -> Set[str]:
 
         :param module_name: `_nodes/ast.ImportFrom.module`.
         :returns: set of importables.
+        :raises ModuleNotFoundError: when we can't find the spec of the `module_name`
+            and/or can't create the module or it's not a standard module.
         """
-        return set(dir(import_module(module_name)))
+        if module_name in pathu.get_standard_lib_names():
+            spec = find_spec(module_name)
+            if spec:
+                module = spec.loader.create_module(spec)  # type: ignore
```

Comment:
this isn't a fix, arbitrary code can still be executed here

Reply:
Pycln now will only handle standard modules, I think the standard ones are safe.

The only change I should do is adding a module path chack.

Reply:
BTW, do you have any alternative solution without losing C star imports?

Reply:
the standard library contains _some stuff_ including test modules which intentionally fault

plus you can override `sys.path` to insert garbage before the standard library

Reply:
up to you to solve it how you want, in my opinion doing dynamic analysis is undesirable and impossible statically (especially in a dynamic language).  I'd scrap the whole "follow * imports" idea entirely

Reply:
Ok, so I'm gonna skip any C star import.

Reply:
fwiw you're going to hit problems with 3rd-party dependencies while running in pre-commit as well, since it expects tools to do static analysis (and isolates them in their own virtualenv) -- additionally why I think dynamic analysis is going to be problematic in other ways

Reply:
I see your point, but actually, Pycln will consider any unimportable module as unexpandable star (it would be skipped).

Pycln can't do only static analysis!
That way a lot of its features would be deprecated, and the differences between it and `unimport` will vanish!

Reply:
Thank a lot for your time!